BTDBTest/ObjectDbTableFreeContentTest.cs

```diff
@@ -746,6 +747,45 @@ static string DumpUnseenKeys(FindUnusedKeysVisitor visitor, string concat)
             return builder.ToString();
         }
 
+        [Fact]
+        public void PossibleToEnumerateRelations()
+        {
+            using (var tr = _db.StartTransaction())
+            {
+                tr.InitRelation<ILinks>("LinksRelation")(tr);
+                tr.InitRelation<ISetings>("SetingsRelation")(tr);
+                tr.InitRelation<IFileTable>("FileRelation")(tr);
+                CheckRelationTypes(tr.EnumerateRelationTypes());
+                tr.Commit();
+            }
+
+            using (var tr = _db.StartTransaction())
+            {
+                CheckRelationTypes(tr.EnumerateRelationTypes());
+            }
+
+            ReopenDb();
+
+            using (var tr = _db.StartTransaction())
+            {
+                tr.InitRelation<ILinks>("LinksRelation")(tr);
+                tr.InitRelation<ISetings>("SetingsRelation")(tr);
+                tr.InitRelation<IFileTable>("FileRelation")(tr);
+                CheckRelationTypes(tr.EnumerateRelationTypes());
+            }
+        }
+
+        void CheckRelationTypes(IEnumerable<Type> types)
```

Comment:
I would move this method as a nested method of the test since its connected to preconditions. Also its better to create expected object (collection) at the beginning of a test and then just assert it. I suppose that the unit test framework supports collection sequential assert.

Reply:
Thank You for the suggestion. Refactored to collection equality assertion.